### PR TITLE
Add initial structure

### PR DESCRIPTION
**Pull request by sirkrypt0**
Open for discussion on the folder structure.

### CODE REVIEW

**Comment by sirkrypt0** [comment]
Maybe it's better to have just one compiled binary for now.

**Comment by frcroth** [comment]
Is it wise to have client and server in the same repo or should we create two repos? I guess one repo will suffice for now. As it is the folder structure is fine.

File: client/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[package]
+name = "client"
+version = "0.1.0"
+authors = ["Tobias <me+github@kantusch.dev>"]
```
Comment (frcroth) [inline client/Cargo.toml]: 😉

**Review by frcroth — APPROVED** [review]

**Comment by frcroth** [comment]
@sirkrypt0 By the way, Clean history or understandable history (Rebases or Merge Commits)?

**Comment by sirkrypt0** [comment]
> @sirkrypt0 By the way, Clean history or understandable history (Rebases or Merge Commits)?

I'm in favor of rebasing.

**Pull request merged**